### DISCUSSION TITLE
How much leverage would you recommend for this strategy?

### DISCUSSION THREAD

**Discussion by al40003** [AlbertoCuadra/algo_trading_weighted_strategy#39] [Q&A] [answered]
How much leverage would you recommend for this strategy?
Looking forward to your strategy at 3commas~

**Comment by AlbertoCuadra** [comment, marked as answer]
Hi @al40003! The current basic version of the algorithm is not suitable for leverage on its own, because backtesting does not take into account possible liquidations due to a high leveraged position and also because entries need to be rationalized. This is a simple backtest in the premium version, where you can buy/sell with limit orders by setting the percentage of new entry positions above 100%. This will mimic the results of leverage if you take into account the maximum drawdown.

**Comment by al40003** [comment]
What year would you recommend to start with?
2017 or 2020
